bucket/arc-welder.json

```
@@ -0,0 +1,17 @@
+{
+    "version": "1.2.0",
+    "description": "Arc welder simplifies gcode for Marlin and Klipper 3D Printer",
+    "license": "LGPL-3.0-or-later",
+    "homepage": "https://github.com/FormerLurker/ArcWelderLib",
+    "url":"https://github.com/FormerLurker/ArcWelderLib/releases/download/1.2.0/Windows.zip",
+    "hash":"CAA17E168CCE43F2C11ED141C405C53082FDFC3E7A162E1649D2EF63A311B612",
```

```suggestion
    "architecture": {
        "64bit": {
            "url":"https://github.com/FormerLurker/ArcWelderLib/releases/download/1.2.0/Windows.zip",
            "hash":"CAA17E168CCE43F2C11ED141C405C53082FDFC3E7A162E1649D2EF63A311B612"
        }
    },
```

The app is built in 64-bit.